src/aktualizr_follower/follower_config.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FOLLOWER_CONFIG_H_
+#define FOLLOWER_CONFIG_H_
+
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
+#include <boost/property_tree/ini_parser.hpp>
+
+struct FollowerNetConfig {
+  int port{9030};
+};
+
+class FollowerConfig {
+ public:
+  FollowerConfig(){};
```

Comment:
Should be space between braces. Do You added this file to clang-format target?

Reply:
I indeed forgot to add the header to the source file checks. But for some reason, clang-format still removes the spaces. I'll look into that.

Reply:
Ok, that's because of the trailing `;` which shouldn't be there. There are a few other instances of these in the code base.